extensions/agent/src/data/pickScheduleData.ts

```diff
@@ -22,10 +22,12 @@ export class PickScheduleData implements IAgentDialogData {
 
 	public async initialize() {
 		let agentService = await AgentUtils.getAgentService();
-		let result = await agentService.getJobSchedules(this.ownerUri);
-		if (result && result.success) {
-			this.schedules = result.schedules;
-		}
+		await agentService.getJobSchedules(this.ownerUri).then((result) => {
+			if (result && result.success) {
```

Comment:
just for my understanding: you want to check for result success or result schedules here?

Reply:
We want to look for success. The schedules can be an empty list.